Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -137,13 +187,7 @@ public extension Array where Element: Equatable {
     /// - Returns: Return array with all duplicate elements removed.
     @discardableResult
     mutating func removeDuplicates() -> [Element] {
-        // Thanks to https://github.com/sairamkotha for improving the method
```

Comment:
Here that would be applicable as optimization:
```swift
mutating func withoutDuplicates() -> [Element] {
      var set: Set<Element> = []
      removeAll(where: { !set.insert($0).inserted }) // Since it makes a partition based removal is more optimized.
      return self
}
```

Reply:
Thank you @LucianoPAlmeida. Added your version to `public extension Array where Element: Hashable { }`.

Reply:
Hello @LucianoPAlmeida 
With `extension Array where Element: Hashable` Codecov reporting that my pull request decreases coverage by some percents. That's why I removed this extension. Should I revert it? Or maybe I should remove `removeDuplicates` and `withoutDuplicates` in `extension Array where Element: Equatable`?